app-text/nfoview/Manifest

```diff
@@ -1 +1 @@
-DIST 1.15.1.tar.gz 91766 SHA256 c0500550978bc20dcc00c4effcb05fe0270868395a499af7e5df3d36a906e47d SHA512 7110de9c8575b057b6b73ecfb2dd1f931007853567f9780c9e90964e4f05e1d0d0d4687bdf5d5911e2514d293c2772035099ee7d3ce6b38adb843dec2f0af1fc WHIRLPOOL 1ba6dcca210fcee72761bea0e0f38c882d8b9f91a6af7e9565b8a86bae2dc7c182a18349549dc642c9607a75f6d05ec8089c3163a06c7ad61d391f00a9d0cbaf
+DIST nfoview-1.15.1.tar.gz 91766 SHA256 c0500550978bc20dcc00c4effcb05fe0270868395a499af7e5df3d36a906e47d SHA512 7110de9c8575b057b6b73ecfb2dd1f931007853567f9780c9e90964e4f05e1d0d0d4687bdf5d5911e2514d293c2772035099ee7d3ce6b38adb843dec2f0af1fc WHIRLPOOL 1ba6dcca210fcee72761bea0e0f38c882d8b9f91a6af7e9565b8a86bae2dc7c182a18349549dc642c9607a75f6d05ec8089c3163a06c7ad61d391f00a9d0cbaf
```

Comment:
Oh my. Squash this into the previous commit. Don't add an issue just to fix it in the same PR.